### ISSUE TITLE
Failing in pulling docs from sync-gateway to couchbase lite

### ISSUE THREAD

**Issue by sridevi-15** [couchbase/sync_gateway#3018]
### Sync Gateway version
sg version 1.4.1-3 and 1.3.1-16

### Operating system
CentOS

### Config file
```{
    "interface":":4984",
    "adminInterface": "0.0.0.0:4985",
    "log": ["CRUD+", "Cache+", "HTTP+", "Changes+"],
    "databases":{
        "db":{


            "server":"http://192.168.33.10:8091",
            "bucket":"data-bucket",
            "username":"data-bucket",
            "password": "password"
        }
    }
}
```

### Log output
https://gist.github.com/sridevi-15/4417802c478b72e2b9414654564272fc

### Steps to reproduce
1.  Setup sync-gateway 1.4.1-3 and CBS 5.0.0-3519 with mac CBL-1.4.1-88
2. Start sync-gateway 
3. Create docs in sync-gateway
4.  Send API to do pull replication from couchbase lite
5. Verify docs shows up on CBL

### Expected behavior
All docs created in sync-gateway should show up on couchbase lite

### Actual behavior
Docs did not show on Couchbase lite

Note : It works fine with sg-accel. Same sg version and CBS version with accel works fine.
It works fine just with sync-gateway with sg version 1.5.0

**Comment by raghusarangapani** [comment]
CBS 5.0.0 does not support TAP, can you try with feed_type="DCP"?

**Comment by sridevi-15** [comment]
thanks @raghusarangapani , tried feed_type and worked fine with sg -1.4.1 ,but still failing with sg-.1.3.1

Here is the sg config of 1.3.1
'''{
    "out.stdout": {
        "adminInterface": "0.0.0.0:4985", 
        "databases": {
            "db": {
                "bucket": "data-bucket", 
                "feed_type": "DCP", 
                "password": "password", 
                "server": "http://s61407cnt72.sc.couchbase.com:8091", 
                "username": "data-bucket"
            }
        }, 
        "interface": ":4984", 
        "log": [
            "CRUD+", 
            "Cache+", 
            "HTTP+", 
            "Changes+"
        ]
    }
}
'''

sg logs are same as what I have already attached

**Comment by sridevi-15** [comment]
@adamcfraser , does DCP supports on sg 1.3.1?

**Comment by adamcfraser** [comment]
@sridevi-15 There were some issues w/ SG 1.3.1 and DCP that didn't get fixed until 1.4.

**Comment by sridevi-15** [comment]
Closing this ticket as sync-gateway 1.3 is not compatible with couchbase server 5.0.0
